Assets/scripts/Player/ControllerPlayer.cs

```diff
@@ -1,28 +1,75 @@
-﻿
-using Interfaces;
+﻿using Interfaces;
+using Interfaces.ServiecesInterface;
+using ObjectPooling;
+using StateMachines;
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
-public class ControllerPlayer:IBasePlayerController
+public class ControllerPlayer : IBasePlayerController
```

Comment:
getting too big?

Reply:
reduced it to 120 by writing single line function in one line.